src/components/shared/Draggable.js

```diff
@@ -15,7 +15,12 @@ export type OnMove = (
 ) => void;
 
 type Props = {
-  value: { +selectionStart: Milliseconds, +selectionEnd: Milliseconds },
```

Comment:
Oh, this is new to me - the generic Draggable component shouldn't be expecting such a specific type. It should be parameterized on the value type instead. I can fix that in a follow-up.

Reply:
Mmm it's not _that_ specific because this would accept any object with these properties. It's not an exact object.

Reply:
That said you're right, the Draggable component doesn't care about its value, it just passes it down.